include/rapidcheck/state/Command.h

```diff
@@ -27,6 +27,8 @@ class Command {
   /// properly.
   virtual void run(const Model &s0, Sut &sut) const;
 
+  virtual std::function<void(const ModelT&)> run(Sut &sut) const;
```

Comment:
I think I also like this variant better. We should document how the relate, though.